libcloud/loadbalancer/drivers/gce.py

```diff
@@ -338,9 +338,12 @@ def _node_to_member(self, node, balancer):
         # would be found if it was there.
         if hasattr(node, 'name'):
             member_id = node.name
-            member_ip = node.public_ips[0]
         else:
             member_id = node
+
```

Comment:
1/ you check if the node object has a public_ips attribute, why? Does it not always have the attribute? If not, we should make sure it always has the attribute.